src/app/components/live-chat/live-chat.component.html

```diff
@@ -1,6 +1,6 @@
 <p-dialog id="live-chat-button"
     header="Live Support Chat"
-    [visible]="display"
+    [(visible)]="display"
```

Comment:
U added () back to visible

Reply:
close button doesnt work if it's only `[visible]` so we will be keeping it as is